utils/enumJsonLists.js

```diff
@@ -0,0 +1,55 @@
+const axios = require('axios');
+const config = require('config');
+
+let portsOfEntry = null;
+let countriesOfResidence = null;
+
+async function fetchPortsOfEntry() {
+  try {
+    const res = await axios.get(`${config.api.url}/api/citizen/ports-of-entry`);
+    portsOfEntry = res.data.map(entry => {
+      entry.value = entry.locationCode;
+      return entry;
+    });
+  } catch (error) {
+    console.log('Error fetching portOfEntry data: ', error);
+  }
+}
+
+function setPortsOfEntry(someList) {
```

Comment:
It looks like you're just exporting this for test purposes. I wouldn't do that, happy to run through alternative approaches.